test/parser_test.cpp

```diff
@@ -0,0 +1,60 @@
+#include <gtest/gtest.h>
+#include <string>
+#include <map>
+#include <fstream>
+#include "../JsonParser.h"
+
+//testing with all data in the string
+TEST(ParserTest, TestString)
```

Comment:
Jó kezdeményezés, hogy vannak sosem használt kulcs-érték párok is.  Viszont jó lenne látni, hogy más típusú hibákkal is boldogul a parzer, mert elvileg fel van készítve ilyesmire.

Reply:
@wajzy
Elkészültem a kért módosításokkal.